src/Css.elm

```diff
@@ -3597,6 +3708,408 @@ alignSelf (Value val) =
+        , calc : Supported
+        }
+    ->
+        Value
+            { num : Supported
+            , zero : Supported
+            , calc : Supported
+            }
+    ->
+        Value
+            { auto : Supported
+            , content : Supported
+            , ch : Supported
+            , cm : Supported
+            , em : Supported
+            , ex : Supported
+            , inches : Supported
+            , mm : Supported
+            , pc : Supported
+            , pct : Supported
+            , pt : Supported
+            , px : Supported
+            , rem : Supported
+            , vh : Supported
+            , vmax : Supported
+            , vmin : Supported
+            , vw : Supported
+            , zero : Supported
+            , calc : Supported
+            }
+    -> Style
+flex3 (Value grow) (Value shrink) (Value basis) =
+    AppendProperty ("flex:" ++ grow ++ " " ++ shrink ++ " " ++ basis)
+
+
+{-| Sets [`flex-direction`](https://css-tricks.com/almanac/properties/f/flex-direction/).
+
+    flexDirection column
+
+-}
+flexDirection :
+    Value
+        { row : Supported
+        , rowReverse : Supported
+        , column : Supported
+        , columnReverse : Supported
+        , inherit : Supported
+        , initial : Supported
+        , unset : Supported
+        }
+    -> Style
+flexDirection (Value val) =
+    AppendProperty ("flex-direction:" ++ val)
+
+
+{-| The `row` [`flex-direction` value](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#Values).
+
+    flexDirection row
+
+-}
+row : Value { provides | row : Supported }
+row =
+    Value "row"
+
+
+{-| The `row-reverse` [`flex-direction` value](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#Values).
+
+    flexDirection rowReverse
+
+-}
+rowReverse : Value { provides | rowReverse : Supported }
+rowReverse =
+    Value "row-reverse"
+
+
+{-| The `column` [`flex-direction` value](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#Values).
+
+    flexDirection column
+
+-}
+column : Value { provides | column : Supported }
+column =
+    Value "column"
+
+
+{-| The `column-reverse` [`flex-direction` value](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction#Values).
+
+    flexDirection columnReverse
+
+-}
+columnReverse : Value { provides | columnReverse : Supported }
+columnReverse =
+    Value "column-reverse"
+
+
+{-| Sets [`flex-wrap`](https://css-tricks.com/almanac/properties/f/flex-wrap/).
+
+    flexWrap wrap
+
+-}
+flexWrap :
+    Value
+        { noWrap : Supported
+        , wrap : Supported
+        , wrapReverse : Supported
+        , inherit : Supported
+        , initial : Supported
+        , unset : Supported
+        }
+    -> Style
+flexWrap (Value val) =
+    AppendProperty ("flex-wrap:" ++ val)
+
+
+{-| The `nowrap` [`flex-wrap` value](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap#Values).
+
+    flexWrap noWrap
+
+-}
+noWrap : Value { provides | noWrap : Supported }
```

Comment:
Awesome work! `nowrap` is all lowercase as a value. Should we be consistent with the values and make this also all lowercase?

Reply:
Actually, `nowrap` already existed for `white-space`, so I removed `noWrap` and added an example in `nowrap`.

Thank you!